backend/src/handlers/index.ts

```diff
@@ -0,0 +1,10 @@
+import { Server, Socket } from "socket.io";
+import registerPlayerHandler, { playerJoin } from "./player.handler";
+
+export { default as Auth } from "./auth.handler";
+
+export const Connection = async (io: Server, socket: Socket): Promise<void> => {
+  registerPlayerHandler(io, socket);
+
+  await playerJoin(io, socket);
+};
```

Comment:
Should this be registering more handlers?

Reply:
Good catch! Fixed in 9654d6a